store/store_test.go

```diff
@@ -471,7 +473,8 @@ func (s *storeTestSuite) TestDownloadRangeRequest(c *C) {
 func (s *storeTestSuite) TestResumeOfCompleted(c *C) {
 	expectedContentStr := "nothing downloaded"
 
-	download = nil
+	// FIXME: hu?
```

Comment:
What?

Reply:
Oh, sorry. Yeah, this one I commented out because I was not sure why it was done like this originally. The comment obviously sucks, I think I need to do a followup for this one.